### DISCUSSION TITLE
DFU on Jumper T-Lite V2

### DISCUSSION THREAD

**Discussion by Shlipez** [EdgeTX/edgetx#5698] [Q&A] [answered]
Please help me, I don't understand how to turn on the DFU on Jumper T-Lite V2, it says on the Internet to turn off the controller, hold down some horizontal buttons, press the on button and turn off the controller, I tried to do this, nothing worked, although a sign appeared on the screen with the inscription: ,,Key stuck."

**Comment by pfeerick** [comment]
No, that is for the bootloader (and it is the inwards horizontal trim
buttons you hold down while turning power on). For the TLite/TLiteV2 you
simply plug the radio in with the power off (and leave it powered off) for
DFU flashing.

https://manual.edgetx.org/edgetx-how-to/access-dfu-and-bootloader-mode

**Reply by Shlipez** [reply]
![image](https://github.com/user-attachments/assets/c264cf7f-b506-4d8a-a33d-11b568c589b4)
There isn't controller to connect

**Reply by Shlipez** [reply]
![image](https://github.com/user-attachments/assets/904dd4df-d7d0-4b30-bb03-dd5db114a0ec)
I tried to do it through openTX, but it still didn't work out.

**Reply by Shlipez** [reply]
![image](https://github.com/user-attachments/assets/aa85111f-9596-40d3-84b7-2ecd82b240f8)
And I tried to do it through edgeTX companion 2.10.5, but it still didn't work out too.

**Reply by pfeerick** [reply, marked as answer]
You're either using a charge only USB cable (some USB leads meant only for charging stuff have the data wires missing) or need the STM32 DFU drivers. i.e. search for the ImpulseRC driver fixer, or install the official STM32CubeProgrammer (which also includes the drivers).

**Reply by Shlipez** [reply]
I have the cable from the original box, it can transmit data, and as for the drivers, I will try to install them, thank you

**Reply by Shlipez** [reply]
Thank you!
